plugins/package-managers/maven/src/main/kotlin/utils/MavenSupport.kt

```diff
@@ -310,6 +313,57 @@ class MavenSupport(private val workspaceReader: WorkspaceReader) {
                 runCatching { Hash(it, algorithm) }.getOrNull()
             } ?: Hash.NONE
 
+        /**
+         * Convert this [RemoteRepository] to a repository in the format used by the Maven Repository System.
+         * Make sure that all relevant properties are set, especially the proxy and authentication.
+         */
+        internal fun RemoteRepository.toArtifactRepository(
+            repositorySystemSession: RepositorySystemSession,
+            repositorySystem: MavenRepositorySystem,
+            id: String
+        ) = repositorySystem.createRepository(url, id, true, null, true, null, null).apply {
+            this@toArtifactRepository.proxy?.let { repoProxy ->
```

Comment:
As the return value does not seem to be used, would `also` be a better fit than `let`?